tests/src/util/runner.ts

```diff
@@ -56,11 +56,32 @@ export class DroneRunner implements DropHandler {
   async certRefresh(
     certs: KeyCertPair,
     natsPort: number,
-    pebble: PebbleResult
+    pebble: PebbleResult,
+    eab = false,
+    eabOptions? : EabOptions
   ) {
+    const eab_cli_options = [
+      "--nats-url",
+      `nats://mytoken@localhost:${natsPort}`,
+      "--https-private-key",
+      certs.privateKeyPath,
+      "--https-certificate",
+      certs.certificatePath,
+      "--cluster-domain",
+      CLUSTER_DOMAIN,
+      "--cert-email",
+      "test@test.com",
+      "--acme-server",
+      `https://localhost:${pebble.port}/dir`,
+      "--acme-eab-key",
+      eabOptions?.key,
+      "--acme-eab-kid",
+      eabOptions?.kid,
+      "cert",
+    ]
     const proc = spawn(
       SPAWNER_PATH,
-      [
+      eab ? eab_cli_options : [
```

Comment:
This is another place where spread notation is preferable to defining two separate args lists.